dashboard/pom.xml

```diff
@@ -4,8 +4,8 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>com.google.cloud.tools.opensource</groupId>
-    <artifactId>cloud-tools-opensource-parent</artifactId>
+    <groupId>com.google.cloud.tools</groupId>
+    <artifactId>cloud-tools-dependencies-parent</artifactId>
```

Comment:
no cloud-tools or dependencies here; maybe just parent

Reply:
As discussed, it's `dependencies-parent`